Source/JavaScriptCore/runtime/JSGlobalObject.cpp

```diff
@@ -1153,7 +1156,6 @@ capitalName ## Constructor* lowerName ## Constructor = featureFlag ? capitalName
 
     if (Options::useSharedArrayBuffer()) {
         putDirectWithoutTransition(vm, vm.propertyNames->SharedArrayBuffer, m_sharedArrayBufferStructure.constructor(this), static_cast<unsigned>(PropertyAttribute::DontEnum));
-        putDirectWithoutTransition(vm, vm.propertyNames->Atomics, atomicsObject, static_cast<unsigned>(PropertyAttribute::DontEnum));
     }
```

Comment:
Let's remove `{` and `}` to align to the WebKit's coding style :)

Reply:
Done. If you're happy with this PR, could you please add the merge queue tag? I'm not yet a committer.